### PR TITLE
[ADD] pos_default_customer: establish a default customer in each POS

### PR DESCRIPTION
**Pull request by jonerikceberio**

### CODE REVIEW

**Comment by rafaelbn** [comment]
Hi again @jonerikceberio 😄 , please could you rebase and check if runbot and travis get green? Thanks!

**Comment by rafaelbn** [comment]
Hi @jonerikceberio I've seen that you have pos repository forked from Eficent/pos and not from OCA/pos so you will get old version of code depending of how ofter Eficent updates its fork



Also take in care that this module is done with all templates, please follow https://github.com/OCA/maintainer-tools/ and undesatand that README is very important, in this PR is totally empty

I will review after all changes are done, thanks

**Review by legalsylvain — APPROVED** [review]
could you set light header in your files ? 
Otherwise LGTM. (code review / no test). thanks ! 
:+1:

File: pos_default_customer/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Point of Sale Default Customer',
+    'version': '8.0.0.1',
+    'category': 'Point Of Sale',
+    'summary': 'Point of Sale Default Customer',
+    'author': 'Digital5 S.L.',
```
Comment (rafaelbn) [inline pos_default_customer/__openerp__.py]: Please review template and OCA
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L10

**Comment by rafaelbn** [comment]
Hi @jonerikceberio are you continuing with this PR?  Thanks

**Comment by jonerikceberio** [comment]
@rafaelbn Done!

**Comment by rafaelbn** [comment]
Thank you @jonerikceberio ! I'm trying to test this PR as the #135 but something is worng with runbot...

While we solve this please could you check travis? There are lint error: https://travis-ci.org/OCA/pos/jobs/371897379

@gurneyalex please do you know why runbot is "pending" for ever in this PR and in https://github.com/OCA/pos/pull/135 ?

I go to https://runbot.odoo-community.org/runbot/repo/github-com-oca-pos-184 and I cannot find #135 

Thank you

**Comment by rafaelbn** [comment]
Hi @jonerikceberio , please could you close this PR and open a new one? Maybe in this way runbot work!

**Comment by legalsylvain** [comment]
FYI : https://github.com/OCA/pos/pull/438

**Pull request closed without merging**